pom.xml

```diff
@@ -40,10 +40,6 @@
       <id>ossrh</id>
       <url>https://s01.oss.sonatype.org/content/repositories/snapshots</url>
     </snapshotRepository>
-    <repository>
```

Comment:
The `nexus-staging-maven-plugin` doesn't need it, but the only way to really test it is to release after this is merged and fix forward if there's an issue with it.